src/parser/jobs/blu/modules/ColdFog.tsx

```diff
@@ -0,0 +1,146 @@
+import {t} from '@lingui/macro'
+import {Plural, Trans} from '@lingui/react'
+import {DataLink} from 'components/ui/DbLink'
+import {Event, Events} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {EventHook} from 'parser/core/Dispatcher'
+import {filter} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {History} from 'parser/core/modules/ActionWindow/History'
+import {Actors} from 'parser/core/modules/Actors'
+import {Data} from 'parser/core/modules/Data'
+import Suggestions, {SEVERITY, TieredSuggestion} from 'parser/core/modules/Suggestions'
+import React from 'react'
+
+// Tiered suggestion severities
+const COLD_SEVERITY = {
+	DROPPED_COLD_FOG: {
+		1: SEVERITY.MAJOR,
+	},
+	DROPPED_WHITE_DEATH_CASTS: {
+		1: SEVERITY.MEDIUM,
+	},
+	INEFFECTIVE_COLD_FOG: {
+		1: SEVERITY.MAJOR,
+	},
+}
+
+const COLD_FOG_MINIMUM_VIABLE_CASTS = 2
+const COLD_COLD_IDEAL_CASTS = 6
+
+interface ColdFogWindow {
+	whiteDeathCasts: number
+}
+
+export class ColdFog extends Analyser {
+	static override handle = 'coldfog'
+	static override title = t('blu.cold_fog.title')`Cold Fog`
+
+	@dependency private actors!: Actors
```

Comment:
Unused dependency, can drop this.

Reply:
Patched in fe8b2e5eeca640fc183bb44db11cc616dd174ba5